File: Tendr/Models/Event.cs
namespace Tendr.Models
{
    public class Event
    {
        public System.Guid Uid { get; set; }

 		public long Id { get; set; }

        public string ConfigCode { get; set; }

        public string StatusCode { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string Url { get; set; }
    }
}

File: src/ui/modules/tendr/models/event.ts
import { IIndexer, Guid } from "@montr-core/models";

export interface IEvent extends IIndexer {
	uid?: Guid;
	id?: number;
	templateUid?: Guid;
	configCode?: string;
	statusCode?: string;
	name?: string;
	description?: string;
}
